allcontainers.js

```diff
@@ -62,6 +63,12 @@ function allContainers (opts) {
     }
 
     switch (data.status) {
+      case 'unpause':
+        if (pausedPreheatContainers[data.id]) {
+          pausedPreheatContainers[data.id] = null;
```

Comment:
Meh, I would rather the perf :D But yes Map is the better solution of all of them :D